engine/src/python/python_compile.cpp

```diff
@@ -99,6 +101,11 @@ PyObject * CompilePython( const std::string &name )
 extern PyObject* PyInit_VS;
 void CompileRunPython( const std::string &filename )
 {
+#if (PY_VERSION_HEX >= 0x03110000)
```

Comment:
Remember, it's `0x030B0000`, not `0x03110000`. :-) That tripped me up at first, too.

Reply:
Yeah, I happened to fall in the same trap by copy-pasting. Very, very bad.

Fixed.

Reply:
No worries. :-) It's looking better now.